backend/src/users/model/dto/userCredentials.dto.ts

```diff
@@ -0,0 +1,9 @@
+import { Length } from "class-validator";
+
+export class UserCredentialsDto {
+	@Length(10, 50)
+	username: string;
+
+	@Length(10, 80)
+	password: string;
```

Comment:
Consider more validation rules for password to force the user to be strong. This could be also frontend stuff but both validation are good.

Reply:
That's on list for do it :)